### PR TITLE
SKR Mini E3 v3.0 definitions

### PR DESCRIPTION
**Pull request by irrenhaus**
Take care: Not yet tested with actual hardware

### CODE REVIEW

**Review by miklschmidt — COMMENTED** [review]
You can't do aliases for the uart_addresses as those aren't pins. Doesn't matter anyway they're only used in this config, so just put them in the [tmc2209] sections 👍

File: boards/btt-skr-mini-e3-30/config.cfg [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
+  x_step_pin=PB13,   x_dir_pin=PB12,   x_enable_pin=PB14,   x_uart_pin=PC11,   x_diag_pin=null,   x_endstop_pin=PC0, x_tx_pin=PC10, x_uart_address=0
+  y_step_pin=PB10,   y_dir_pin=PB2,   y_enable_pin=PB11,   y_uart_pin=PC11,   y_diag_pin=null,   y_endstop_pin=PC1, y_tx_pin=PC10, y_uart_address=2
+  z0_step_pin=PB0,  z0_dir_pin=PC5,  z0_enable_pin=PB1,  z0_uart_pin=PC11,  z0_diag_pin=null,   z_endstop_pin=PC2, z_tx_pin=PC10, z_uart_address=1
   z1_step_pin=null,  z1_dir_pin=null,  z1_enable_pin=null,  z1_uart_pin=null,  z1_diag_pin=null,
   z2_step_pin=null,  z2_dir_pin=null,  z2_enable_pin=null,  z2_uart_pin=null,  z2_diag_pin=null,
-  e_step_pin=PB3,   e_dir_pin=PB4,   e_enable_pin=PD1,   e_uart_pin=PC11,   e_diag_pin=null,   e_heater_pin=PC8,  e_sensor_pin=PA0, e_tx_pin=PC10,
+  e_step_pin=PB3,   e_dir_pin=PB4,   e_enable_pin=PD1,   e_uart_pin=PC11,   e_diag_pin=null,   e_heater_pin=PC8,  e_sensor_pin=PA0,  e_tx_pin=PC10, e_uart_address=3,
 # accel
-  adxl345_cs_pin=null,
+  adxl345_cs_pin=PD9, # Take care: shared with the SD card
```
Comment (miklschmidt) [inline boards/btt-skr-mini-e3-30/config.cfg:15]: True, that means we need to use software_spi_* in the [adxl345] section below.

**Comment by irrenhaus** [comment]
@miklschmidt This now is how I am running it with my printer.

**Comment by miklschmidt** [comment]
Implemented in the development branch, thanks!

**Pull request closed without merging**